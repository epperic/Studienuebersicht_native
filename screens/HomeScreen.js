import React from "react";
import { Image, View } from "react-native";
import { styles } from "../styles";

class HomeScreen extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
      image: require('../assets/Stundenplan.png')
    }
  }

  render() {
    return (
      <View style={styles.container}>
        <Image source={this.state.image} style={{width: '100%'}} />
      </View>
    );
  }
}

export default HomeScreen;