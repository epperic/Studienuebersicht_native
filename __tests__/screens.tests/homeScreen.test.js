
import 'react-native';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import HomeScreen from '../../screens/HomeScreen';

describe('HomeScreen', () => {
  it('renders correctly', () => {
    const homeScreen = renderer.create(
      <HomeScreen />
    ).toJSON();
    expect(homeScreen).toMatchSnapshot();
  });
});