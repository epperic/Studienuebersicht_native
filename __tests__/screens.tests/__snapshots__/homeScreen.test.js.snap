// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen renders correctly 1`] = `
<View
  style={
    Object {
      "flexGrow": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source="test-file-stub"
    style={
      Object {
        "width": "100%",
      }
    }
  />
</View>
`;
